Revise README

lmullen opened the pull request:
@dselivanov This is a revision of the README along the lines of what I did for the vignettes.

Some suggestions:

- Should the links to the tutorials be replace with links to or instructions on how to get to the vignettes? Seems like the vignettes are less likely to be out of date.
- You have a discussion of the Corpus object under "Terminology." I think you could cut the whole terminology section since it is out of date for 0.3 and since it's too much information for a README.

Going to work on function level documentation next.

lmullen commented:
Oh, and this will fix a NOTE in R CMD check about a non-standard file in the package root.

dselivanov commented on README.md:
```diff
@@ -0,0 +1,44 @@
+
+The API is built around the [iterator](https://en.wikipedia.org/wiki/Iterator) abstraction. The API is concise, providing only a few functions which do their job well. The package does not (and probably will not in the future) provide trivial very high-level functions. But other packages can build on top of the framework that text2vec provides.
+
+## Terminology 
```
can delete whole section,  I copied all the gory technical details to [wiki](https://github.com/dselivanov/text2vec/wiki)

dselivanov commented on README.md:
```diff
@@ -0,0 +1,44 @@
+[![Downloads](http://cranlogs.r-pkg.org/badges/grand-total/text2vec)](http://cran.rstudio.com/package=text2vec)
+[![Follow](https://img.shields.io/twitter/follow/dselvan0v.svg?style=social)](https://twitter.com/intent/follow?screen_name=dselvan0v)
+
+# Tutorials
```
We can replace links with CRAN-based html version of vignettes (as you suggested):
[GloVe](https://cran.r-project.org/web/packages/text2vec/vignettes/glove.html), [vectorization](https://cran.r-project.org/web/packages/text2vec/vignettes/text-vectorization.html)
Also will be nice to add link to articles marked with [text2vec tag](http://dsnotes.com/tags/text2vec/).

dselivanov commented:
Thanks for the PR!
Totally agree with suggestions. Can you please adjust the PR (see inlined comments)?
Also don't forget to add yourself to contributors :-)

lmullen commented:
Okay, I put links to the vignettes, deleted that text, fixed a few more things I noticed, and added myself as a contributor. I think that's all for this PR.
